source/utils/sentenceHelper.py

```diff
@@ -11,16 +11,24 @@
 FULL_WIDTH_TERMINATORS_PATTERN: re.Pattern = re.compile("[。！？]")
 
 
-def findEndOfSentence(s: str, offset: int) -> int:
+def findEndOfSentence(sentence: str, offset: int) -> int:
```

Comment:
This should be marked private

```suggestion
def _findEndOfSentence(sentence: str, offset: int) -> int:
```